lib/dialogs/add-comment.js

```diff
@@ -0,0 +1,38 @@
+
+module.exports = (message, channel, team) => {
+  const linkToMessage =
+    `https://${team.domain}.slack.com/archives/${channel.id}/p${message.ts.replace('.', '')}`;
+
+  const comment = `> ${message.text}\n<sub>[View message in Slack](${linkToMessage})</sub>`;
+
+  return {
+    callback_id: 'add-comment',
+    title: 'Comment on Issue or PR',
```

Comment:
I know we had a long discussion about not using `PR`, but don't remember the conclusion. Could this be `Comment on GitHub`?

Reply:
I remember a couple of discussions around this 😅 The one where I remember the outcome was about whether to use "PR" or "Pull request" and it was that we need to use "PR" because "Pull request" is too long. Slack wold not show the dialog :( https://api.slack.com/dialogs

![image](https://user-images.githubusercontent.com/7718702/41598848-60283b1a-73c9-11e8-884c-6ab219582cba.png)



There are a couple of very tight restrictions like this around dialogs unfortunately